<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>364</width>
    <height>729</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="record_mode">
    <property name="geometry">
     <rect>
      <x>163</x>
      <y>20</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>רציף</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>בגילוי בלבד</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>מוד הקלטה</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>80</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>FPS</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="fps_box">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>80</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>9</number>
    </property>
    <property name="value">
     <number>9</number>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_roi_btn">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>470</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>אפס איזור עניין</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>690</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>שמור שינויים</string>
    </property>
   </widget>
   <widget class="QWidget" name="vid_holder" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>320</width>
      <height>240</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>295</x>
      <y>110</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>שם WIFI</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>295</x>
      <y>140</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>סיסמה</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ssid_txt">
    <property name="geometry">
     <rect>
      <x>145</x>
      <y>110</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>8</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="pw_txt">
    <property name="geometry">
     <rect>
      <x>145</x>
      <y>140</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>8</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="wifi_interval_box">
    <property name="geometry">
     <rect>
      <x>206</x>
      <y>170</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>60</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>170</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>מרווחי סריקה</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>288</x>
      <y>510</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ערך גילוי:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>550</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>סף גלאי:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="limit_txt">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>550</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="metric_txt">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>510</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="vid_state">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>גלאי פעיל</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>50</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>זמן  הקלטה</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="rec_time">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>50</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>30</number>
    </property>
    <property name="value">
     <number>5</number>
    </property>
   </widget>
   <widget class="QPushButton" name="left_btn">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>640</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>שמאלה</string>
    </property>
   </widget>
   <widget class="QPushButton" name="right_btn">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>640</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>ימינה</string>
    </property>
   </widget>
   <widget class="QSlider" name="positionslide">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>610</y>
      <width>160</width>
      <height>19</height>
     </rect>
    </property>
    <property name="maximum">
     <number>180</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>90</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="position_scroll">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>590</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>צידוד</string>
    </property>
   </widget>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
